### DISCUSSION TITLE
Segwit P2SH addresses

### DISCUSSION THREAD

**Discussion by wrkerr** [comit-network/xmr-btc-swap#788] [Q&A] [answered]
My bitcoin wallet uses Segwit bech32 P2SH addresses to accommodate a multi-signature setup (https://blog.muun.com/muuns-multisig-model/). UnstoppableSwap.net appears not to support this address format.
![image](https://user-images.githubusercontent.com/8811716/134031316-30941297-c014-4175-9a1d-ea5159ea5762.png)
Could this be added?

**Comment by binarybaron** [comment, marked as answer]
UnstoppableSwap.net is 3rd party website and not related to this project.
